dist/windows/installer.nsi

```diff
@@ -78,7 +78,7 @@ Section $(inst_startmenu) ;"Create Start Menu Shortcut"
 
   CreateDirectory "$SMPROGRAMS\qBittorrent"
   CreateShortCut "$SMPROGRAMS\qBittorrent\qBittorrent.lnk" "$INSTDIR\qbittorrent.exe"
-  CreateShortCut "$SMPROGRAMS\qBittorrent\Uninstall.lnk" "$INSTDIR\uninst.exe"
+  CreateShortCut "$SMPROGRAMS\qBittorrent\$(inst_uninstall_link_description)" "$INSTDIR\uninst.exe"
```

Comment:
Shouldn't it be `"$SMPROGRAMS\qBittorrent\$(inst_uninstall_link_description).lnk"`?

Reply:
It's. true- Sorry.